client/js/lib/d3.js

```diff
@@ -136,15 +137,18 @@ module.exports = function ({
 		.attr('stop-opacity', 1);
 
 	const rootNode = svg.select('.rootNode');
+	const baseColor = ringColor ||  "#fff1e0";
+	debugger;
+	const dampener = 1.1;
 	for (let i=0; i<rings.length; i++) {
 		rootNode.append('circle')
 			.attr('class', 'background')
 			.attr('r', (rings[i].max + 1) * ringSize)
-			.style('fill', rings[i].fill);
+			.style('fill', baseColor);
```

Comment:
It won't fallback to a solid color, it will fall back to an FT color that will gradually get darker.